helm/prometheus-rules/templates/alerting-rules/falco.rules.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+    cluster_type: "management_cluster"
+  name: falco.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: falco
+    rules:
+    - alert: FalcoCriticalAlertFiring
+      annotations:
+        description: '{{`Pod {{ $labels.k8s_ns_name }}/{{ $labels.k8s_pod_name }} triggered the Falco rule {{ $labels.rule }}.`}}'
+      expr: increase(falco_events{priority=~"0|1|2|3"}[10m] ) > 0
```

Comment:
Open to suggestions from those more knowledgeable. Falco events are atomic, so there were tradeoffs getting to this expression.
Originally, the idea was to page on any non-zero event count (with the optimistic goal that we get to 0 false positives on MCs). The downside there is that the alert fires forever once it fires once and "new" events occurring later are not meaningfully distinguished from the "old" ones.
Using `increase()` has the downside that it could miss events outside the sliding window (e.g. if prom is down) and that alerts will self-close after some time no matter what.

Is there a better way? Ultimately, we probably won't use the existing prom + alertmanager setup for Falco alerting in the long term, but we'll do this for now.

Reply:
Maybe @giantswarm/team-atlas has a better idea?